src/target/riscv_debug.c

```diff
@@ -918,6 +927,36 @@ static const char *riscv_fpu_ext_string(const uint32_t extensions)
 	return "";
 }
 
+/*
+ * Generate the fpu section of the description
+ * fpu_size =32 single precision float
+ * fpu_size =64 double precision float
+ */
+static size_t riscv_build_target_fpu_description(size_t fpu_size, char *const buffer, size_t max_length)
```

Comment:
Given the order of parameters on the base function, the compiler will likely generate a smaller amount of code for the call frame if `fpu_size` is made the last parameter. This is fine otherwise.